mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMessageManager.java

```diff
@@ -538,6 +522,27 @@ public MetaData getMetaData(boolean resetRecent, MailboxSession mailboxSession,
         return new MailboxMetaData(recent, permanentFlags, uidValidity, uidNext, highestModSeq, messageCount, unseenCount, firstUnseen, isWriteable(mailboxSession), isModSeqPermanent(mailboxSession), resolvedAcl);
     }
 
+    private MetaData sensibleInformationFreeMetaData(MailboxSession mailboxSession, MailboxACL resolvedAcl) throws MailboxException {
```

Comment:
This seems to be even possible to make it a static constant in MailboxMetaData class (or at least a static method of this class)